sklearn/feature_extraction/text.py

```diff
@@ -909,6 +911,10 @@ def _make_int_array():
     """Construct an array.array of a type suitable for scipy.sparse indices."""
     return array.array(str("i"))
```

Comment:
> I get TypeError: must be char, not unicode if the str() is not there..

Interesting, which platform and version of python ? It works fine for me for all the python versions I tried (2.6, 2.7, 3.3, 3.4, 3.5). Python installed through conda on a Ubuntu box, if that matters.